examples/shared_samples/server_profile.rb

```diff
@@ -71,6 +71,7 @@
 item3.retrieve!
 puts "\nServer Profile updated successfully! Name: #{item3['name']}"
 
+# This method supports till OneView REST API Version 1200
```

Comment:
still begin and rescue block is missing here.

Reply:
@soodpr its there. Unavailable method for these methods are added to library class for SP for API1600 and begin rescue is added in example file.

Reply:
its not present at line# 75. Please add it

Reply:
@soodpr Its added now.